assets/js/modules/pagespeed-insights/datastore/__fixtures__/desktop-report.json

```diff
@@ -0,0 +1,5 @@
+{
+    "report": {
+        "data": "PageSpeed Insights Desktop Report Data"
+    }
+}
```

Comment:
Fixtures should be specific to the datapoint they request, with any relevant variation as part of it; i.e. `pagespeed--desktop.json`.

Apart from that, this isn't an actual response from the API so it doesn't offer much value as a fixture. We will need real data for these requests to use in Storybook as soon as the components are refactored so we may as well add it here.

Please rename these fixtures and fill them with the response from the API, using `https://example.com/` as the `url`, one for each strategy. You can do this with the plugin by setting your custom site URL in the dev settings and clearing your local+session storage so it makes the requests again.

Reply:
Ah ok, I had done something like that originally, but removed it as we weren't actually testing anything within those responses.